viewing_party/party.py

```diff
@@ -1,23 +1,174 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+
+    #Checks if any of the dictionary keys are None
+    if title == None or genre == None or rating == None:
+        return None
+
+    movie = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+
+    return movie

```

Comment:
Given the relative simplicity of this function, I'd be okay with the following: 
 
```suggestion
    return {
        "title": title,
        "genre": genre,
        "rating": rating
    }
```